### DISCUSSION TITLE
How Does This Service Make Money To Support It's Servers?

### DISCUSSION THREAD

**Discussion by sancho1952007** [deta/cloud-docs#629] [General] [answered]
I Am Really Shocked To See A Company Provide Such Insanely Cool Services For Free.
But, I Wonder How Do You All Manage Your Server Costs Giving Free Service To All Your Users?

**Comment by AyushSehrawat** [comment, marked as answer]
https://basic.deta.dev/faq/#why-are-they-free-how-do-they-make-money-how-are-they-planning-to-sustain

**Reply by sancho1952007** [reply]
Thanks for the info 👍
